README.md

```diff
@@ -13,7 +13,25 @@ This library provides a Ruby client to connect with the PayPro API.
 
 ## Installation
 
-We provide an installation guide on https://paypro.nl/developers/api-clients/ruby.
+Installation is pretty standard:
+
+```sh
+$ gem install paypro
+```
+
+if you'd rather install PayPro using `bundler` you can put this in your Gemfile:
```

Comment:
MIsschien: If you'd rather install the PayPro client using `bundler` you can put this in your Gemfile: